### DISCUSSION TITLE
so spindle coupling

### DISCUSSION THREAD

**Discussion by grid1025** [raphaelvallat/yasa#118] [Q&A] [answered]
The so sigma coupling can be done easily by yasa, but most papers are interested in so spindle coupling. How can I conduct it by yasa.

**Comment by raphaelvallat** [comment, marked as answer]
I actually do think that SO—sigma is more relevant, because spindle detection relies on arbitrary thresholds and as a result two scorers scoring the same night would probably not agree on what is or what isn't a spindle. Furthermore, there are actually very few SO events that occur in combination with a spindle (10-20% max). That said, it is possible to find co-occurring SO and spindles using this function: https://raphaelvallat.com/yasa/build/html/generated/yasa.SWResults.html#yasa.SWResults.find_cooccurring_spindles
